### DISCUSSION TITLE
PCD Printing

### DISCUSSION THREAD

**Discussion by partisani** [hneemann/Digital#946] [Q&A]
I'm building a simple processor and i need to print it later, but it will have embedded circuits and some exclusive things, so, how do i do it without recreating it entirely on a site **(but i don't even know a site for that)**

So, i need a way to export it in a valid way that the site will understand, and the site

**Thanks for reading!**

**Comment by hneemann** [comment]
If you want to make a PCB, I can recommend using KiCAD, but there is no export from Digital to KiCAD.
After you have successfully simulated the circuit, you will have to redraw the circuit in KiCAD.

**Reply by partisani** [reply]
but what about all of the embedded circuits? it will have a lot of them, if KiCAD doesn't support embedded circuits it will be a lot harder.
